backend/routes/projectRoutes.js

```diff
@@ -1,6 +1,10 @@
 const router = require("express").Router();
-const { projectCtrl } = require("../controllers")
+const { projectCtrl } = require("../controllers");
 
 // TODO ROUTES GO HERE
+router.get('/', projectCtrl.getProject);
+router.post("/", projectCtrl.createProject);
+router.put('/:id', projectCtrl.updateProject);
+router.delete('/:id', projectCtrl.deleteProject);
 
```

Comment:
import isAuthenticated from the middleware index and put it on all of these except get